src/passwordPenalty.js

```diff
@@ -1,8 +1,11 @@
 export default function penaltyPoints(password) {
+  if (password === null) return 0;
   if (typeof password !== "string") password = String(password);
+  //regex to return array of duplicate values
   const REG_EXP = /(.)\1+/g;
-  if (password === "null") return 0;
+ 
   const MATHCES = password.match(REG_EXP);
   if (!MATHCES) return 0;
+  
   return MATHCES.reduce((count, cur) => count + (cur.length > 2 ? 2 : 1), 0);
```

Comment:
You can check if this is not a typo error.
If it is not you can make it more descriptive because it's very hard to understand what Mathces mean here.